lighthouse-core/audits/unsized-images.js

```diff
@@ -67,31 +67,34 @@ class SizedImages extends Audit {
    * @param {LH.Artifacts.ImageElement} image
    * @return {boolean}
    */
-  static isUnsizedImage(image) {
+  static isSizedImage(image) {
     const attrWidth = image.attributeWidth;
     const attrHeight = image.attributeHeight;
     const cssWidth = image.cssWidth;
     const cssHeight = image.cssHeight;
-    const widthIsValidAttribute = SizedImages.isValidAttr(attrWidth);
-    const widthIsValidCss = SizedImages.isValidCss(cssWidth);
-    const heightIsValidAttribute = SizedImages.isValidAttr(attrHeight);
-    const heightIsValidCss = SizedImages.isValidCss(cssHeight);
+    const widthIsValidAttribute = UnsizedImages.isValidAttr(attrWidth);
+    const widthIsValidCss = UnsizedImages.isValidCss(cssWidth);
+    const heightIsValidAttribute = UnsizedImages.isValidAttr(attrHeight);
+    const heightIsValidCss = UnsizedImages.isValidCss(cssHeight);
     const validWidth = widthIsValidAttribute || widthIsValidCss;
     const validHeight = heightIsValidAttribute || heightIsValidCss;
-    return !validWidth || !validHeight;
+    return validWidth && validHeight;
   }
 
   /**
    * @param {LH.Artifacts} artifacts
    * @return {Promise<LH.Audit.Product>}
    */
   static async audit(artifacts) {
-    // CSS background-images are ignored for this audit.
-    const images = artifacts.ImageElements.filter(el => !el.isCss);
+    // CSS background-images & ShadowRoot images are ignored for this audit.
+    const images = artifacts.ImageElements.filter(el => !el.isCss && !el.isInShadowDOM);
     const unsizedImages = [];
 
     for (const image of images) {
-      if (!SizedImages.isUnsizedImage(image)) continue;
+      const isFixedImage =
+      image.cssComputedPosition === 'fixed' || image.cssComputedPosition === 'absolute';
```

Comment:
nit (should add an indentation level when breaking up an assignment statement)
```suggestion
        image.cssComputedPosition === 'fixed' || image.cssComputedPosition === 'absolute';
```